meson.build

```diff
@@ -203,6 +203,7 @@ if get_option('default_library') != 'static'
 endif
 
 if get_option('default_library') == 'static'
+    glibconfig_conf.set('GLIB_DYNAMIC_UNLOADING', '1')
```

Comment:
We could do this unconditionally, as it's a no-op when building against a dynamic GLib.